spring-kafka/src/main/java/org/springframework/kafka/support/KafkaHeaders.java

```diff
@@ -67,4 +68,14 @@
 	 */
 	public static final String RECEIVED_PARTITION_ID = PREFIX + "receivedPartitionId";
 
+	/**
+	 * The header for holding the {@link org.apache.kafka.common.record.TimestampType type} of timestamp.
+	 */
+	public static final String TIMESTAMP_TYPE = PREFIX + "timestampType";
+
+	/**
+	 * The header for holding the timestamp of the record.
+	 */
+	public static final String TIMESTAMP = PREFIX + "timestamp";
```

Comment:
Oh, no issues @artembilan, keep the review coming :-), will make this change and update the PR